desktop/src/main/java/bisq/desktop/primary/main/left/LeftNavView.java

```diff
@@ -306,20 +306,31 @@ private VBox createSubmenu(LeftNavSubButton... items) {
     private void setupButtonHandler(NavigationTarget navigationTarget, LeftNavButton button) {
         button.setOnAction(() -> {
             controller.onNavigationTargetSelected(navigationTarget);
+            if (button.isHasSubmenu()) {
+                // Always expand menu when header button is clicked
+                button.getIsSubMenuExpanded().set(true);
+            }
             maybeAnimateMark();
         });
         controller.onNavigationButtonCreated(button);
     }
 
+    private void expandCollapseHandler(NavigationTarget navigationTarget) {
+        if (model.getSelectedNavigationButton().get() == null) {
+            // There is no button selected on startup
+            controller.onNavigationTargetSelected(navigationTarget);
+        }
+        maybeAnimateMark();
+    }
+
     private void maybeAnimateMark() {
         LeftNavButton selectedLeftNavButton = model.getSelectedNavigationButton().get();
         UIThread.runOnNextRenderFrame(() -> {
             double targetY = menuTop + selectedLeftNavButton.getBoundsInParent().getMinY();
             if (selectedLeftNavButton instanceof LeftNavSubButton) {
                 for (int i = 0; i < mainMenuItems.getChildren().size(); i++) {
                     Node item = mainMenuItems.getChildren().get(i);
-                    if (item instanceof VBox) {
-                        VBox submenu = (VBox) item;
+                    if (item instanceof VBox submenu) {
```

Comment:
As stated in the other PRs commont we want to stick with Java 11 featureset for Android support.